### PR TITLE
Adds new recipe: Risotto.

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

I realized while playing cook the other day that we did not have risotto as a recipe, I was very disappointed to see we did not have it.

## Why It's Good For The Game

Risotto is good, therefore it is good for the game. Our rice recipes are quite dull so hopefully, this will be a neat new thing to do with it. 

Sprites borrowed/stolen from Aurora <https://github.com/Aurorastation/Aurora.3/pull/2567>
image

## Changelog
:cl: carshalash, Sprites borrowed/stolen from Aurora's Synnono
add: Risotto is now a cookable recipe in the misc category.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
add risotto to the list of wanted foods by italian bots

**Comment by carshalash** [comment]
Hi bebe.

**Comment by carshalash** [comment]
> add risotto to the list of wanted foods by italian bots

I think that should work?

**Review by Twaticus — APPROVED** [review]

**Comment by tralezab** [comment]
Risotto, a salad?

File: code/game/objects/items/food/salad.dm [hunk 1]
```diff
@@ -101,6 +101,15 @@
 	tastes = list("rice" = 1, "meat" = 1)
 	foodtypes = GRAIN | MEAT
 
+/obj/item/food/salad/risotto
+	name = "risotto"
+	desc = "Proof the Italians mastered every carb."
+	icon_state = "risotto"
```
Comment (ATH1909) [inline code/game/objects/items/food/salad.dm:107]: if we took the sprites from polaris, shouldn't we credit them somewhere?

Comment (Fikou) [reply code/game/objects/items/food/salad.dm:107]: theyre on the pr

**Comment by carshalash** [comment]
> Risotto, a salad?

Yeah... apparently all rice dishes are classified as salads. It's dumb and probably needs fixing soon™

**Comment by Farquaar** [comment]
"I felt full after the risotto."

**Pull request merged**